File: DSA/Assignment_3/sparse.c
#include <stdio.h>
int main()
{
    int m,n;
    printf("Enter row and colummn size:");
    scanf("%d %d",&m,&n);
    int arr[m][n];
    for(int  i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("Enter element:");
            scanf("%d",&arr[i][j]);
        }
    }
    printf("Matrix:\n");
    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("%d ",arr[i][j]);
        }
        printf("\n");
    }

    for(int i=0;i<m;i++)
    
    return 0;
}

File: DSA/Assignment_3/sparse.py
m=int(input("Enter row size:"))
n=int(input("Enter column size:"))
arr=[]
for i in range(m):
    a=[]
    for j in range(n):
        a.append(int(input("Enter element:")))
    arr.append(a)

c=0
for i in range(m):
    for j in range(n):
        if arr[i][j]==0:
            c+=1

if c>m*n/2:
    print("Sprase Matrix")
else:
    print()